plugins/woocommerce/client/blocks/tests/e2e/tests/templates/constants.ts

```diff
@@ -126,7 +126,7 @@ export const CUSTOMIZABLE_WC_TEMPLATES: TemplateCustomizationTest[] = [
 
 			await page.goto( '/product/wordpress-pennant/' );
 		},
-		templateName: 'External Product Add to Cart with Options',
+		templateName: 'External Product Add To Cart With Options',
```

Comment:
Is there a particular reason for this change? Or just for consistency?

Reply:
I updated the template search logic which resulted in case-sensitive matching. The original name is uppercased so I had to update it in the constant there.